docs/guides/operations-guide/openstack/cinder.md

```diff
@@ -39,3 +39,20 @@ dragon@mgr001:~$ openstack --os-cloud admin volume service list
 | cinder-scheduler | testbed-node-2                    | internal | enabled | up    | 2023-10-01T08:56:25.000000 |
 +------------------+-----------------------------------+----------+---------+-------+----------------------------+
 ```
+
+## Sync quota
+
+It can happen that more block storage usage is stored in the database for a project than
+is actually used. This can be corrected using [cinder-manage](https://docs.openstack.org/cinder/latest/cli/cinder-manage.html).
+
+For all projects:
+
+```
+$ docker exec -it cinder_api cinder-manage quota sync
+```
+
+Only for a specific project:
+
+```
+$ docker exec -it cinder_api cinder-manage quota sync --project-id PROJECT_ID
+```
```

Comment:
Also mention the `quota check` command which does a dry-run of the above?

I'm also not sure yet whether using the cinder_api container is a good choice, it leads to the logs of the command appearing within the `cinder-api.log` file. otoh having a dedicated container might be overdoing things?